ilastik/pixels_classification.ipynb

```diff
@@ -228,6 +228,8 @@
     }
    ],
    "source": [
+    "os.environ[\"LAZYFLOW_THREADS\"] = \"2\"\n",
+    "os.environ[\"LAZYFLOW_TOTAL_RAM_MB\"] = \"1000\"\n",
```

Comment:
deployed these two lines manually into the notebook. Now, the notebook went through the whole Blastocysts dataset without a hitch.